scripts/download-extensions

```diff
@@ -49,13 +49,32 @@ createext() {
     fi
 }
 
+# In the event that kernel-rt is out of sync with vanilla kernel, determine
+# the right version of kernel-headers to use.  Assumes that `createext()`
+# has successfully created/downloaded the kernel-rt extension
+get_rtheaders() {
```

Comment:
Nice, I think we'll want something similar for the regular `kernel-headers` BTW (but instead taking the EVR from the commit metadata). I have a card for that, so can do that in a follow-up. :)

Reply:
SGTM.  I knew you had that card in your queue, so I wanted to get this in front first.

(Of course, after I made this PR I realized I should have added a check to see if the EVR matched so we could noop this function)

Reply:
Ugh, this is causing the RHCOS 4.7 pipeline to fail

`ln: failed to create hard link '../kernel-headers-4.18.0-240.1.1.el8_3.x86_64.rpm': File exists`

Reply:
https://github.com/openshift/os/pull/450